compiler/dyno/include/chpl/parsing/parsing-queries.h

```diff
@@ -147,6 +148,18 @@ uast::Function::ReturnIntent idToFnReturnIntent(Context* context, ID id);
  */
 bool functionWithIdHasWhere(Context* context, ID id);
 
+void setConfigSettings(Context* context, ConfigSettingsList keys);
+
+const
+ConfigSettingsList& configSettings(Context* context);
+
+
+// note the ID to which we have associated a config set on the compile command-line
+void
```

Comment:
I think that this should be file-private and not declared in this header, since realistically the builder is the only thing that calls this (?). However this exposes a big can of worms which is that in order for `nameToConfigSettingId` to produce interesting/"correct" results, the parsing query has to be called first. Maybe that's not a problem, though?

Reply:
@arezaii and I talked about moving these as private methods within Builder.cpp but I don't know why that didn't make it to the version you reviewed. Maybe because of the error checking in config.cpp—if that's the case, we need to add a function it can call that is implemented within Builder.cpp as I said in https://github.com/chapel-lang/chapel/pull/19643/files#r854224717

Reply:
Sounds great, thanks!